recipes/csvmonkey/all/conanfile.py

```diff
@@ -1,7 +1,8 @@
 import os
 
-from conans.errors import ConanInvalidConfiguration
-from conans import ConanFile, CMake, tools
+from conan.errors import ConanInvalidConfiguration
+from conan import ConanFile, tools
+from conans import CMake
```

Comment:
```suggestion
```

Reply:
It will lead to a fail. Not everything can be automated

Reply:
it won't fail, cmake is not used in the recipe.